docs/unified/novacustom/camera-switch.md

```diff
@@ -0,0 +1,9 @@
+# Camera switch setting
+
+Novacustom laptops have the option to disable the webcam completely. After
+disabling this option, the power to the camera is completely cut off.
+
+## Changing setting
+
+You can find **Enable Camera** option in **Dasharo System Features** in submenu
+**Dasharo Security Option**. By default the option is enabled.
```

Comment:
```suggestion
**Dasharo Security Options**. By default the option is enabled.
```

Reply:
fixed in 42563eee50bae817d0b20418e1b73d34775acafc